bit_manipulation/binary_count_trailing_zeros.py

```diff
@@ -0,0 +1,43 @@
+from math import log2
+
+
+def binary_count_trailing_zeros(a: int):
```

Comment:
```suggestion
def binary_count_trailing_zeros(a: int) -> int:
```

Reply:
@dhruvmanila I have done the required changes!